gordo_components/cli/workflow_generator.py

```diff
@@ -230,13 +230,14 @@ def unique_tag_list_cli(machine_config: str, output_file_tag_list: str):
     machines = NormalizedConfig(yaml_content, project_name="test-proj-name").machines
 
     tag_list = set(tag for machine in machines for tag in machine.dataset.tag_list)
+
     if output_file_tag_list:
         with open(output_file_tag_list, "w") as output_file:
-            for item in tag_list:
-                output_file.write("%s\n" % item)
+            for tag in tag_list:
+                output_file.write("%s\n" % tag.name)
```

Comment:
Should we change this to a f-string, as it seems like we overall lean towards using them?
 ```
output_file.write(f"{tag.name}\n")
```